libraries/RC_Channel/RC_Channel.h

```diff
@@ -559,6 +559,9 @@ class RC_Channels {
     // returns true if we have had a direct detach RC reciever, does not include overrides
     bool has_had_rc_receiver() const { return _has_had_rc_receiver; }
 
+    // returns true if we have had a overrides on any channel
```

Comment:
I guess this comment should be "if we have had an override on any channel"?

Reply:
fixed